wolfssl/wolfcrypt/settings.h

```diff
@@ -27,6 +27,8 @@
 #ifndef WOLF_CRYPT_SETTINGS_H
 #define WOLF_CRYPT_SETTINGS_H
 
+#include <wolfssl/options.h>
```

Comment:
This file is only generated when `./configure` is used to build. We could include a blank options.h template. We could also include this in the case when `#ifndef WOLFSSL_USER_SETTINGS`, but I would want to see this down below with the include "user_settings.h". I'm still not sure this is acceptable 100% of the time, but we will review.